pkgs/development/tools/dstp/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGo117Module, fetchFromGitHub }:
+
+buildGo117Module rec {
+  pname = "dstp";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "ycd";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1pxzaz2a261lsnbdbr9km1214a4jzq2wgkdfvf9g966gsa4nqfl6";
+  };
+
+  vendorSha256 = "1n1kx4zcskndikjl44vwmckr6x5cv6cacwdwfwjjsf6aqgagpld8";
+
+  doCheck = false;
```

Comment:
Why are tests disabled?

Reply:
Tests are doing request to known public urls (since this is a tool to perform network diagnostics). I added a comment to clarify that